src/silo/query_engine/filter_expressions/expression.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <string>
 
+#include <re2/re2.h>
```

Comment:
Why is this include needed *here*?

Reply:
This is related to your point raised in #536, this include is needed for the `make_unique` of a `StringSearch` object. As the header file only forward declares the re2, to which it owns a unique_ptr, we need to be able to resolve a `sizeof(RE2)` in this file. This is changed including `<re2/re2.h>` in `string_search.h`